components/edit-collective/VirtualCard.js

```diff
@@ -259,14 +259,14 @@ const VirtualCard = props => {
             <Flex>
               <Box mt="19px" mr={4}>
                 <P fontSize="18px" fontWeight="700" lineHeight="26px">
-                  {props.privateData.expireDate}
+                  {props.privateData.expiryDate}
 
-                  <Action color="black" ml={2} onClick={handleCopy(props.privateData.expireDate)}>
+                  <Action color="black" ml={2} onClick={handleCopy(props.privateData.expiryDate)}>
                     <Copy size="18px" />
                   </Action>
                 </P>
                 <P fontSize="12px" fontWeight="500" lineHeight="16px">
-                  <FormattedMessage id="VirtualCards.ExpireDate" defaultMessage="MM/YYYY" />{' '}
+                  <FormattedMessage id="VirtualCards.ExpiryDate" defaultMessage="MM/YYYY" />{' '}
```

Comment:
Let's maybe not rename this one, it's not a big deal if we keep the `id` like this, and changing it would force translators to redo the work for every language.

Reply:
I updated all the translation files. I don't understand why the translators would have to redo something?

Reply:
Ok I just figured out what you meant. I ran `npm run langs:update` and copy pasted the original translation to the new field in the translation files.

Reply:
Unfortunately, the tool we use, https://crowdin.com/project/opencollective, will detect the change in the original string and will reset it for all other languages after we merge this PR. It's the desired behavior as changes on a string usually need to be applied to other languages as well, but it doesn't play well in those situations where we're only touching the `id`.

Reply:
I reverted the change on the id